File: Android App - Shredsheets 2/app/src/main/java/revert/shredsheets/enums/Instruments.java
package revert.shredsheets.enums;

public enum Instruments {
    Guitar6(0, "Guitar"), Guitar7(1, "7 String Guitar"), Bass4(2,"Bass"), Bass5(3,"5 String Bass"), Ukulele(4,"Ukulele");

    private int instrumentId;
    public static Instruments[] OrderedKeys = new Instruments[]{Instruments.Guitar6, Instruments.Guitar7, Instruments.Bass4, Instruments.Bass5, Instruments.Ukulele};

    private String cleanName;
    public String getCleanName()
    {
        return cleanName;
    }

    private Instruments(int instrumentId, String cleanName) {
        this.instrumentId = instrumentId;
        this.cleanName = cleanName;
    }

    public int getValue() {
        return instrumentId;
    }
}

File: Shredsheets 3/core/src/com/bm/shredsheets/enums/Instruments.kt
package com.bm.shredsheets.enums

enum class Instruments(val value: Int, val cleanName: String) {
    Guitar6(0, "Guitar"), Guitar7(1, "7 String Guitar"), Guitar8(2, "8 String Guitar"), Bass4(3, "Bass"), Bass5(4, "5 String Bass"), Ukulele(5, "Ukulele");

    companion object {
        var OrderedKeys = arrayOf(Guitar6, Guitar7, Guitar8, Bass4, Bass5, Ukulele)
    }
}
